### ISSUE TITLE
Multiple notebook selection

### ISSUE THREAD

**Issue by slobotai1** [att/rcloud#1186]
It would be nice if the user can select multiple notebooks for deleting, forking or staring.
Also user should be able to perform the above actions on a complete folder of notebooks.
This will really help a lot
Thanks.

**Comment by gordonwoodhull** [comment]
Yes it would be nice. One complication is that for now, selecting a notebook opens it. It might be confusing to have an interface where the first click opens a notebook but later shift-clicks just add to the selection.

The folder commands are easier, #716 and #1149 cover these.
